packages/grpc-js/src/server.ts

```diff
@@ -334,65 +370,61 @@ export class Server {
 
   private getChannelzSessionInfoGetter(
```

Comment:
With this modified signature, this method is more accurately named `getChannelzSessionInfo`.